test/files/run/t8928/Array_0.java

```diff
@@ -0,0 +1,15 @@
+package test;
+
+import java.lang.annotation.*;
+
+@Retention(RetentionPolicy.RUNTIME)
+//TODO: make this work
+//@java.lang.annotation.Repeatable(Array_0.Repeated.class)
```

Comment:
Did I write this test? That was so long ago.

I did ["make this work"](https://github.com/scala/scala/pull/6846), btw. Not sure if the test actually needs that case.